crime-commons-classes/src/main/java/uk/gov/justice/laa/crime/util/DateUtil.java

```diff
@@ -48,6 +54,18 @@ public static LocalDateTime convertDateToDateTime(LocalDate date) {
 
     public static ZonedDateTime toZonedDateTime(LocalDateTime localDateTime) {
         return (localDateTime == null) ? null :
-        ZonedDateTime.parse(localDateTime.atZone(ZoneId.systemDefault()).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
+                ZonedDateTime.parse(localDateTime.atZone(ZoneId.systemDefault()).format(DateTimeFormatter.ISO_OFFSET_DATE_TIME));
+    }
+
+    public static LocalDateTime stringToLocalDateTime(String dateString, String dateFormat) {
+        if (dateString != null) {
+            try {
+                return LocalDateTime.ofInstant(DateUtils.parseDate(dateString, dateFormat).toInstant(), ZoneId.systemDefault());
```

Comment:
Might have been an idea to pass in a ZoneId for the correct time as there have been timezone issues between our services?

For example if one of our services used this as UTC then another BST it could cause a one hour difference. I guess this is a general issue in this Util class.